src/adapters/controllers/auth/Auth.controller.ts

```diff
@@ -1,11 +1,13 @@
 import { ICredentials } from "../../../application/modules/auth/dtos/Credentials.dto";
 import { TokenDto } from "../../../application/modules/auth/dtos/TokenDto";
 import { IServiceContainer } from "../../shared/kernel";
-import container, { LoginUseCase } from "./container";
+import container, { LogoutUseCase, LoginUseCase } from "./container";

```

Comment:
Please, I know it's an stupid desire, but locate the imports from longest to shortest in order to have a beauty code.  
This comment also apply to many files.